python/torch_mlir_e2e_test/test_suite/conv.py

```diff
@@ -737,48 +759,10 @@ def __init__(self):
     def forward(self, inputVec):
         return torch._C._nn.upsample_nearest2d(inputVec,
                                                output_size=[8, 11],
-                                               scale_factors=None)
+                                               scales_h=None,
+                                               scales_w=None)
 
 
 @register_test_case(module_factory=lambda: UpSampleNearest2dDiffSize())
 def UpSampleNearest2dDynamicSize_basic(module, tu: TestUtils):
     module.forward(tu.rand(2, 3, 2, 2))
-
-
-class UpSampleNearest2dDiffFactor(torch.nn.Module):
-
-    def __init__(self):
-        super().__init__()
-
-    @export
-    @annotate_args([None, ([-1, -1, -1, -1], torch.float32, True)])
-    def forward(self, inputVec):
-        return torch._C._nn.upsample_nearest2d(inputVec,
-                                               output_size=None,
-                                               scale_factors=[2.3, 4.7])
-
-
-@register_test_case(module_factory=lambda: UpSampleNearest2dDiffFactor())
-def UpSampleNearest2dDynamicFactor_basic(module, tu: TestUtils):
-    module.forward(tu.rand(2, 3, 2, 2))
```

Comment:
What's the reason to remove this test?

Reply:
`output_size` is not OPTIONAL anymore so these tests aren't really valid anymore.
Let me know your take here.

Reply:
Okay, I'm looking for a test with scaling factors similar to this.

Reply:
I've addressed this - you may review again.

Reply:
Have added `UpSampleNearest2dDynamicFactor_basic` to `EAGER_MODE_XFAIL_SET`.